libert_util/include/ert/util/log.h

```diff
@@ -26,12 +26,23 @@ extern "C" {
 #include <stdbool.h>
 #include <stdlib.h>
 
+//Same as pythons default log levels, with numeric values equal to python / 10.
```

Comment:
I set them to python/10 to keep a bit of compability with the old levels, which is not defined anywhere, but it seems like 1 is used a lot, and that is also the default level. I am not able to see far enough in to the future to determine if it is better to change those and standarize on the same numeric values as python uses.